src/main/java/seedu/address/logic/commands/FavouriteCommand.java

```diff
@@ -68,6 +72,22 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(String.format(MESSAGE_FAVOURITE_PERSON_SUCCESS, editedPerson));
     }
 
+    @Override
+    public CommandResult executeInDetailedView(Model model) {
+        requireNonNull(model);
+
+        Person personToFavourite = model.getDetailedContactViewPerson();
+        Favourite newFavouriteStatus = personToFavourite.getFavouriteStatus().equals(Favourite.IS_FAVOURITE)
+                ? Favourite.NOT_FAVOURITE
+                : Favourite.IS_FAVOURITE;
```

Comment:
I'm not sure if this was copied from my work (if it is I'm sorry), but a better way to do this might be to do `personToFavourite.isFavourite() ? NOT_FAVOURITE : IS_FAVOURITE`

Reply:
Haha yes 😅 I'll change all occurrences of it.